### ISSUE TITLE
Not an issue - but

### ISSUE THREAD

**Issue by avianion** [NeoWzk/alicrawler#1]
Can this be used to upload an image and do reverse image search like on the aliexpress mobile application?

**Comment by NeoWzk** [comment]
no, to upload image to aliexpress, will need to use API from aliexpress, you can optimize this basic crawler and add the upload function in it
